src/iceth-client/src/lib.rs

```diff
@@ -112,7 +129,7 @@ impl Client {
     /// Sends a raw transaction to the external EVM.
     pub async fn send_raw_transaction(&self, transaction_bytes: Bytes) -> Result<H256, Error> {
         let data = format!(
-            r#"{{"jsonrpc":"2.0","id":"3","method":"eth_sendRawTransaction","params":["0x{transaction_bytes:x}"]}}"#,
+            r#"{{"jsonrpc":"2.0","id":4,"method":"eth_sendRawTransaction","params":["0x{transaction_bytes:x}"]}}"#,
```

Comment:
why is the id changed? does it has any particular meaning in this scope?

Reply:
No, I've tried to keep id unique for each call.